htop.c

```diff
@@ -46,17 +46,18 @@ static void printHelpFlag(void) {
    fputs("htop " VERSION "\n"
          COPYRIGHT "\n"
          "Released under the GNU GPLv2.\n\n"
-         "-C --no-color               Use a monochrome color scheme\n"
-         "-d --delay=DELAY            Set the delay between updates, in tenths of seconds\n"
-         "-F --filter=FILTER          Show only the commands matching the given filter\n"
-         "-h --help                   Print this help screen\n"
-         "-M --no-mouse               Disable the mouse\n"
-         "-p --pid=PID,[,PID,PID...]  Show only the given PIDs\n"
-         "-s --sort-key=COLUMN        Sort by COLUMN (try --sort-key=help for a list)\n"
-         "-t --tree                   Show the tree view by default\n"
-         "-u --user[=USERNAME]        Show only processes for a given user (or $USER)\n"
-         "-U --no-unicode             Do not use unicode but plain ASCII\n"
-         "-V --version                Print version info\n"
+         "-C --no-color                   Use a monochrome color scheme\n"
+         "-d --delay=DELAY                Set the delay between updates, in tenths of seconds\n"
+         "-F --filter=FILTER              Show only the commands matching the given filter\n"
+         "-h --help                       Print this help screen\n"
+         "-M --no-mouse                   Disable the mouse\n"
+         "-p --pid=PID,[,PID,PID...]      Show only the given PIDs\n"
```

Comment:
Minor unrelated typo:

Should be `--pid=PID[,PID,PID...]` (no comma before brackets)